### ISSUE TITLE
HDR not recognized with RTX HDR on Steam Deck

### ISSUE THREAD

**Issue by Delamain813** [LeGeRyChEeSe/Sunshine-AIO#24]
I run Moonlight with the HDR box ticked and when I start up Kingdome Come Deliverance, which I usually play using RTX HDR from the NVIDIA App, the HDR filter is not available to be activated, which suggest that HDR isn't actually working. 

The option is greyed out, just like it is when I forget to activate HDR on my regular TV before starting the game. 

Also, when I just go into the NVIDIA App to monitors, it doesn't actually seem to recognize that HDR is really active. 

I'm pretty sure this is something that changed, as I recall having no HDR related issues with the verion of Sunshine AIO before 2.0. 

Otherwise I have no issues, it runs great.

**Comment by LeGeRyChEeSe** [comment]
are you running on Windows 10 ? Because VDD for W10 doesn’t support HDR natively.

**Comment by Delamain813** [comment]
No, I'm running Windows 11. Idk why, but inside the Nvidia App, it does't register that HDR theoretically is enabled.

**Comment by LeGeRyChEeSe** [comment]
Have a look in the Windows Display Settings and see if HDR is activated. Otherwise you should have installed the wrong version of VDD. To know which version of VDD you installed go to the Device Manager and if it's not IddSampleDriver Device HDR then you should uninstall it and install the VDD version for Windows 11 manually as the script is now broken to install VDD automatically due to recent releases from there.
